mRides-app/ChatsActivity.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+
+namespace mRides_app
+{
+    [Activity(Label = "Chats")]
+    public class Chats : Activity
+    {
+        protected override void OnCreate(Bundle savedInstanceState)
+        {
+            base.OnCreate(savedInstanceState);
+
+            // Create your application here
```

Comment:
This is a deprecated file that was removed from the project but somehow did not get deleted WIll delete